en/datatypes/ref.adoc

```diff
@@ -0,0 +1,225 @@
+= Ref! datatype
+:toc:
+:numbered:
+
+== Abstract
+
+`ref!` values denote named references, such as social media handlers or munged e-mail addresses with an implied domain name.
+
+`ref!` datatype belongs to `any-string!` typeset while being conceptually close to an `issue!` datatype, differing from it in implementation (string-like vs. symbol-like) and offered set of trade-offs.
+
+== Creation
+
+`ref!` values can be created using literal syntax at load-time or constructed at run-time using either a `make` constructor or a `to` conversion.
+
+=== Literal syntax
+
+`ref!` values support only one unique literal form described below.
+
+*Syntax*
+
+----
+<ref> format : @<name>
+
+<name> : optional name of a referent
+----
+
+* `<name>` is zero or more characters other than `# $ ' , = > @ ^`;
+* literal form is terminated by one of the `" ( ) ; < [ ] { }` or blank (space, tab, `CR`, `LF`) characters.
+
+*Examples*
+
+----
+@red_lang
+Nenad Rakocevic @ Dockimbel
```

Comment:
Somewhat confusing. Better to have single examples, one per line. For an empty ref, include a comment.